### PR TITLE
Add seconds in time picker

### PR DESCRIPTION
**Pull request by baig**
- [x] I have changed my target branch to **develop** :facepunch: 

## Description
Adds a Seconds view in the time picker.

time-picker-with-seconds

time-picker-with-seconds-without-ampm

### CODE REVIEW

**Comment by dmtrKovalenko** [comment]
Wow looks good. :)

**Comment by cherniavskii** [comment]
I'm wondering - what is the use-case for that?

**Comment by dmtrKovalenko** [comment]
@cherniavskii really there is. I know several people that using this pickers for timers. And they are need for seconds.

**Comment by cherniavskii** [comment]
@dmtrKovalenko okay, but still I can't imagine any particular use-case for seconds in time picker.
Can you give an example?

File: lib/src/TimePicker/TimePicker.jsx [hunk 1]
```diff
@@ -20,26 +22,29 @@ export class TimePicker extends Component {
...
     isHourViewShown: true,
+    isMinuteViewShown: false,
     meridiemMode: this.props.utils.getHours(this.props.date) >= 12 ? 'pm' : 'am',
   }
 
   setMeridiemMode = mode => () => {
     this.setState(
       { meridiemMode: mode },
-      () => this.handleChange(this.props.date, false, false),
+      () => this.handleChange(this.props.date, false, false, false),
```
[the remark this answers is not in the document]
Comment (baig) [reply lib/src/TimePicker/TimePicker.jsx]: Right, I have pushed the requested changes.

**Comment by baig** [comment]
@dmtrKovalenko The use cases are all those situations where you want to allow users to select a "precise" time point.

**Pull request merged**